pkg/controller/storage/account/account.go

```diff
@@ -187,7 +187,7 @@ func newAccountSyncDeleter(ao azurestorage.AccountOperations, kube client.Client
 
 func (asd *accountSyncDeleter) delete(ctx context.Context) (reconcile.Result, error) {
 	asd.acct.Status.SetConditions(runtimev1alpha1.Deleting())
-	if asd.acct.Spec.ReclaimPolicy == runtimev1alpha1.ReclaimDelete {
+	if asd.acct.Spec.DeletionPolicy == runtimev1alpha1.DeletionDelete {
```

Comment:
Do we need to handle the case where the DeletionPolicy is not set (and should default to delete, iirc)?

Reply:
Now it deleted if the policy is empty.